src/containers/CallIn/ThankYou/ThankYou.js

```diff
@@ -93,7 +100,7 @@ export class ThankYou extends Component {
     }
 
     reportCall = (trackingToken, callerId, calledDistrict) => {
-        const reportBody = trackingToken ? {
+        const reportBody = trackingToken && callerId ? {
```

Comment:
add breadcrumb for the case where caller id is included but not tracking (and vice versa)